docs/aqua-book/language/values.md

```diff
@@ -123,9 +122,78 @@ func foo(a: i32, b: i32) -> i32:
   -- remainder
   e = g % f
   
-  -- Can use arithmetics anywhere
-  -- Better use brackets to enforce ordering
-  <- (a + b) - (c + d * (e - 6 ))
+  -- It is possible to use arithmetics
+  -- anywhere an integer is required
+  <- a + (b - c) + d * (e - 6)
+```
+
+Precedence of operators from highest to lowest:
+- `**`
+- `*`, `/`, `%` (left associative)
+- `+`, `-` (left associative)
+
+## Comparison operators
```

Comment:
Aqua also supports `==` for any values, maybe mention that?

Reply:
For now it supports it only inside `if` and it is mentioned in docs on `if`

Reply:
Well, you have section title as `Comparison operators`. `==` is surely one of them.

Reply:
Added a note